backend/src/main/java/com/example/backend/service/WorkTypeService.java

```diff
@@ -0,0 +1,20 @@
+package com.example.backend.service;
```

Comment:
Your pr looks great. I have also seen that you have implemented integration tests. Just one question. Was it the intention to generate the integration test for the service class? Integration tests are very recoursenintentive with the @SpringBootTest annotation and are mostly generated for the controller to test the integration of the different dependencies in actual application environment. As the controllers usually do not contain any business logic, it is sufficient to write just a few tests. 
The service classes, on the other hand, usually contain the business logics that we want to test in their own functionality within the class. Here we want to consider and test all possible cases. If possible, without dependencies on other classes. We achieve this by not executing the service tests in the application environment (i.e. without @SpringBootTest annotation), but by mocking the dependencies.
I hope this explanation was helpful.